### ISSUE TITLE
Newsletter? 

### ISSUE THREAD

**Issue by bethschechter** [maptime/maptime-admin#51]
Is this something we want? If so, who wants to do it?

**Comment by kkowalsky** [comment]
think we can use the xyHT blog series as this? I think it'd work nicely as one in the same?

**Comment by bethschechter** [comment]
Works for me. The main goal is for people to be writing about what's going
on at their chapters :)

//typed with thumbs

**Comment by sgoulet** [comment]
Hi @bethschechter @kkowalsky ... I believe I saw a sign up sheet for blogging for xyHT somewhere possibly in a Google doc?  I might've been dreaming however :sleeping:.  If it exists, possibly link it to the [resources page](http://maptime.io/lessons-resources/) under a new hashtag for this kind of outreach?  Maybe not there, but somewhere central maybe, possibly?  Cc: @lyzidiamond et al

**Comment by bethschechter** [comment]
@sgoulet, yes! It's here
https://github.com/maptime/maptime-admin/wiki/xyHt-Monthly-Sign-up

Hopefully the Wiki is just as easy as Google Docs :)

**Comment by sgoulet** [comment]
Thanks @bethschechter.  The wiki so easy! :ok_hand:

**Comment by bethschechter** [comment]
great! that's what I love to hear :)

**Comment by kkowalsky** [comment]
Things I think we should include:
1. Announcements
2. Maptime in the News (if applicable)
3. Applause section (I love the idea of hearing about the cool things we're doing!)
4. Highlight on one Maptime chapter (rotating)
5. Events calendar (I think we should also have an event calendar on the website?)

**Comment by geobrando** [comment]
> Events calendar (I think we should also have an event calendar on the website?)

See PR waiting for review in maptime/maptime.github.io#192

**Comment by bethschechter** [comment]
@katiekowalsky I love this idea. ...how do you feel about running with it?
or do have too much on your plate as it is?

**Comment by kkowalsky** [comment]
Yeah, I think it'll be consolidating a lot of info so I don't mind! I'll take sometime on this first one, see how I feel about it (just thought of it: popular tweets to include)

**Comment by bethschechter** [comment]
sounds great, Katie! thank you!

**Comment by kkowalsky** [comment]
Hey @geobrando or anyone, is there a way we can speed up the event calendar on the website? I'm not sure how i'll assemble the events section in the maptime newsletter w/o it

**Comment by geobrando** [comment]
Hi @kkowalsky.  I can merge but we should get someone from HQ to sign off on it first.

But note that this calendar will only be populated once organizers add their events via a google form which will look something like [this](https://docs.google.com/forms/d/1OfA8sV0Vs0-WlCwRBALC8p_RR1zMFS-WXYtXnyDzTA0/viewform). So the website calendar won't be very useful in the short term.

**Comment by bethschechter** [comment]
Ahoy @geobrando and @kkowalsky: part of the reason the calendar hasn't been
implemented is just that: how do we make it easy for people to get their
events on there? If we had everything on Eventbrite or Meetup that would be
one thing; we could scrape and pull. But if it's a manual process then I'm
not sure how well this would be maintained.

Any advice for the easiest workflow here would be super appreciated!

**Comment by geobrando** [comment]
@bethschechter You're absolutely right. I expect a pretty low participation rate as most won't want to bother with both creating a meetup/eventbrite event and filling out a google form.  Pulling directly is the ultimate answer and I think @rhewitt22's solution is just a temporary one.

So the question for you is, would you prefer a website calendar that's missing the majority of events or none at all? We could absolutely use the Eventbrite and Meetup APIs to get most of the events but I'm not sure if anyone currently has the bandwidth to tackle that.  It doesn't matter to me either way.

**Comment by bethschechter** [comment]
Ok - wonkadoo workaround that I just discovered just now.

Goal: Katie needs dates for the newsletter
Temp solution: use Meetup and Eventbrite calendar features to find what
people are listing.

Here's Meetup
<http://www.meetup.com/find/events/?allMeetups=false&keywords=maptime&radius=Infinity&userFreeform=San+Francisco%2C+CA&mcId=z94110&mcName=San+Francisco%2C+CA&eventFilter=mysugg>.
Search for Maptime in any distance, go to calendar feature (button on
mid-upper right), find all upcoming Maptimes. Boom!

Eventbrite
<https://www.eventbrite.com/d/canada/maptime/?crt=regular&sort=best&vp_ne_lat=68.0772&vp_ne_lng=-5.6983&vp_sw_lat=-57.7923&vp_sw_lng=-170.9327>
is a little less straightforward. You have to search by map, which you
can't find until after you've hit Search on the main page. When you get to
a page that looks like this:

[image: Inline image 1]

Then search by map in the upper left. You'll need to do a few queries to
get the whole globe.

Is it a perfect system? No, but it will work for now (I hope!). Perhaps we
can find a more elegant query for the future to get this info onto the
site.

LMK if any of this doesn't make sense.

Thanks!
Beth

@bethschechter
Maptime <http://maptime.io/> founder
ATL –> SF –> PDX

**Comment by geobrando** [comment]
:+1: 

Since some don't use either, Kaite can go through the list at http://maptime.io/chapters and look for chapters without a Meetup icon.  For those that link to a website or twitter, check those out some create events directly on their blog or advertise through twitter and some chapters link to other event sites.  Shouldn't be too many to go through.

**Comment by bethschechter** [comment]
Ultimately this is why a calendar would be nice - to slurp up these events
one day!

@bethschechter
Maptime <http://maptime.io/> founder
ATL –> SF –> PDX

**Comment by kkowalsky** [comment]
Yeah, unfortunately I think I'll just have to do that. While a calendar is awesome and great, it's not feasible right now.

**Comment by bethschechter** [comment]
I RECKON I DONE GOT ME AN IDEAR.

@kkowalsky what if, instead of the newsletter, you take on doing xYht posts about individual chapters each month? You select the chapter you want to talk about, and then (a) they can write it themselves or (b) you can write it? We can also set up a time limit for this, like 6 months, and then hand it off to someone when the time is up. 

Just trying to think of ways to keep this simple - you are the only one to sign up so far. I think this is easier than trying to pass the buck for a new writer each month. 

@lyzidiamond @almccon @camilleanne would love your thoughts

**Comment by almccon** [comment]
Totally. Brilliant.

**Comment by bethschechter** [comment]
thanks @almccon 
:)

**Comment by kkowalsky** [comment]
hmmmm, do you mean like basically organize the xyHT article for each month? I'm not sure I could handle having to write each article if that's what it ends up being... Explain more.

**Comment by bethschechter** [comment]
Copy that. I think that's what I was thinking, in lieu of the newsletter.
If it makes it harder, then screw it. That's not what I'm going for :)

**Comment by kkowalsky** [comment]
Let me come up with a sample newsletter in the next week and see what it's like. I want to make sure that: 
1. This process isn't too complicated for finding upcoming maptimes
2. How often people want these/if there's enough of an interest (can we have organizers do a google form or something?)
3. What sections we should be featuring

**Comment by bethschechter** [comment]
awesome. Thank you, Katie!

**Comment by kkowalsky** [comment]
should we put a subscribe button on our website? How are we going to have people subscribe? Should I add everyone already in the chapters spreadsheet?

**Comment by almccon** [comment]
We also have the dormant maptime-organizers and maptime-announce Google groups. I think we could unilaterally populate the newsletter list with the membership of those two groups. 

Or, we just say that maptime-announce IS the newsletter list, and have people sign up there if they want to receive the newsletter.

**Comment by geobrando** [comment]
Having a hard time following but I think I get it.  This will be a regular email and something separate from the blog series, right?

Using one of the google groups definitely makes sense, as there's an easy mechanism for subscribing and unsubscribing.  What is the audience for this? Is it organizers or all maptimers?

Something of interest:
http://www.nptechforgood.com/2014/09/28/10-e-newsletter-best-practices-for-nonprofits/

**Comment by kkowalsky** [comment]
@almccon i think that's a good idea of doing that. The goal of the newsletter is to inform/announce any news so it's the basic principle of maptime-announce.

@geobrando: yes. It's going to include general maptime news (for instance: Maptime summit or the new bylaws) that HQ wants people to know. It'll also have a monthly format with brief tweets, photos, and  stories from different chapters (TBD but hopefully along those lines).
